mmv1/templates/terraform/decoders/spanner_instance_config.go.erb

```diff
@@ -0,0 +1,14 @@
+config := meta.(*transport_tpg.Config)
+d.SetId(res["name"].(string))
```

Comment:
this is needed because API response has FQDN example "project/{}/instanceConfigs/{name}" I want to abstract that out and only expose "name"

Reply:
Another reason is because [google_spanner_instance](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/spanner_instance) expects only name, by this one can use variable from output of this in google_spanner_instance.